styles.css

```
@@ -0,0 +1,92 @@
+body {
+  background-color: #333;
+  color: #7fff00;
+  font-family: "Lucida Console", Monaco, monospace;
+}
+
+a {
+  color: #66cd00;
+}
+
+a:hover {
+  color: #ffd700;
+}
+
+nav>ul {
```

Espacios alrededor de `>`
```suggestion
nav > ul {
```